studio/components/interfaces/SQLEditor/SQLEditor.tsx

```diff
@@ -299,6 +301,16 @@ const SQLEditor = () => {
         ])
       }
 
+      Telemetry.sendEvent(
+        {
+          category: 'sql_editor',
+          action: 'ai_suggestion_accept',
```

Comment:
sorry my bad. Actions should be past tense. So gotta change to `ai_suggestion_accepted`